sdk/program/src/vote/state/mod.rs

```diff
@@ -88,6 +89,30 @@ impl Lockout {
     }
 }
 
+#[derive(Serialize, Default, Deserialize, Debug, PartialEq, Eq, Copy, Clone, AbiExample)]
+pub struct LandedVote {
+    // Latency is the difference in slot number between the slot that was voted on (lockout.slot) and the slot in
+    // which the vote that added this Lockout landed.  For votes which were cast before versions of the validator
+    // software which recorded vote latencies, latency is recorded as 0.
+    pub latency: u8,
+    pub lockout: Lockout,
+}
+
+impl From<LandedVote> for Lockout {
+    fn from(landed_vote: LandedVote) -> Self {
+        landed_vote.lockout
+    }
+}
+
+impl From<Lockout> for LandedVote {
+    fn from(lockout: Lockout) -> Self {
+        Self {
+            latency: 0,
+            lockout,
+        }
+    }
+}
+
```

Comment:
```suggestion

impl LandedVote {
    pub fn slot(&self) -> Slot {
        self.lockout.slot
    }
    
    pub fn confirmation_count(&self) -> u32 {
        self.lockout.confirmation_count
    }
}
```

see https://github.com/solana-labs/solana/pull/29753.  now all of the noisy plumbing can go away

Reply:
This is fine; although I personally think that automatic conversions like this are on balance not  beneficial, as they don't mean anything in isolation and thus defeat readability in places where I think being explicit leads to better self-documentation of code.  But I'm definitely not going to argue with anyone at Solana Labs about this stuff, I'm just your lackey at this point.

Reply:
it's a code maintenance advantage.  going to get rid of a bunch of noise here and ease review